packages/checkout-ui-extensions-run/README.md

```diff
@@ -0,0 +1,74 @@
+# `checkout-ui-extensions-run`
+
+This package provides a command line interface for developing and building Checkout UI extensions.
+
+## Installation
+
+```bash
+$ yarn add @shopify/checkout-ui-extensions-run --dev
+```
+
+## Usage
+
+This package will add a `checkout-ui-extensions-run` binary to your project. You can execute this binary directly, or through yarn:
+
+```bash
+$ node_modules/.bin/checkout-ui-extensions-run <command>
+$ yarn checkout-ui-extensions-run <command>
+```
+
+### `checkout-ui-extensions-run dev`
+
+> alias: `checkout-ui-extensions-run serve`
+
+```bash
+$ yarn checkout-ui-extensions-run dev
+```
+
+This command will start a local development server for your extension. This development server, based on [`webpack-dev-middleware`](https://github.com/webpack/webpack-dev-middleware), will serve the JavaScript file for your extension, and will recompile it as you make changes to your codebase.
+
+This command will print the `localhost` address of the asset server when it starts up. To control the port the server will bind to, you can pass the `--port` flag:
+
+```bash
+$ yarn checkout-ui-extensions-run dev --port 3000
+```
+
+If there is a website that can load a preview of your extension, this command can automatically navigate your browser to that page on start. To do so, pass a `--open` flag with the URL to navigate to:
+
+```bash
+$ yarn checkout-ui-extensions-run dev --open https://argogogo.dev
+```
+
+This command will open your default browser to the `open` URL, with an `extension` query parameter set to a JSON-stringified version of the extension asset’s URL. Additionally, if you pass an `--extension-point` flag to this command, it will be forwarded as an `extension-point` query parameter when the URL is opened in a browser.
+
+The most complex example might look something like this:
+
+```bash
+$ yarn checkout-ui-extensions-run dev --open https://argogogo.dev --extension-point Checkout::PostPurchase::ShouldRender
+```
+
+Which would open your default browser to a the following URL, assuming the default asset server port is available:
+
+```
+https://argogogo.dev?extension=%2522http%253A%252F%252Flocalhost%253A8910%252Fassets%252Fextension.js%2522&extension-point=Checkout::PostPurchase::ShouldRender
```

Comment:
We should remove all the references to `argogogo`. We'll probably remove the `--open` flag since I don't think it's really useful anymore; I think we can safely just remove the documentation for it.